packages/console/src/pages/Connectors/utils/index.ts

```diff
@@ -43,3 +44,14 @@ export const getConnectorGroups = <
     });
   }, []);
 };
+
+export const getConnectorGroupPlaceholder = (type: ConnectorType): ConnectorGroup => ({
+  id: nanoid(),
+  connectors: [],
+  type,
+  name: { en: '' },
+  description: { en: '' },
+  logo: '',
+  logoDark: null,
+  target: '',
+});
```

Comment:
Should it be const configs instead of a util method?

Reply:
Updated.